data_override/include/data_override.inc

```diff
@@ -399,6 +412,20 @@ subroutine read_table(data_table)
              read(record,*,err=99) data_entry%gridname, data_entry%fieldname_code, data_entry%fieldname_file, &
                                    data_entry%file_name, data_entry%interpol_method, data_entry%factor, region, &
                                  & region_type
+
+             if (index(data_entry%file_name, ":") .ne. 0) then
+               data_entry%multifile = .true.
+               index_1col = index(data_entry%file_name, ":")
+               index_2col = index(data_entry%file_name(index_1col+1:), ":")
+               data_entry%prev_file_name = data_entry%file_name(1:index_1col-1)
+               data_entry%next_file_name = data_entry%file_name(index_1col+index_2col+1:)
+               ! once previous/next files are filled in, overwrite current
+               data_entry%file_name = data_entry%file_name(index_1col+1:index_1col+index_2col-1)
```

Comment:
we could test for 2 occurences of  the `:` separator between files to assure there are either 1 or 3 files but not 2. This would bulletproof the code I agree, good catch!